contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -558,6 +560,52 @@ def sync(self, pk, attributes=False, tags=False, files=False, assessment_items=F
                 progress_tracker=progress_tracker,
             )
 
+    def deploy_from_changes(self, changes):
+        errors = []
+        for deploy in changes:
+            try:
+                self.deploy(self.request.user, deploy["key"])
+            except Exception as e:
+                log_sync_exception(e, user=self.request.user, change=deploy)
+                deploy["errors"] = [str(e)]
+                errors.append(deploy)
+        return 1
```

Comment:
This should return `errors`-- I'm getting `'int' object is not subscriptable` errors when testing

Reply:
resolved! thanks for catching this.